### PR TITLE
Virus spiked skin (Cornu Cutaneum)

### LINKED ISSUE #754: [BOUNTY]: Viro Additions+Changes: Spiked Skin
**What this specific bounty entails:**
Code the following virus symptom, following the description and symptom thresholds:
Spiked Skin
Hurts people when they grab you, hug you, or you walk into them or hug them.
Stealth: -3
Resistance: 3
Speed: -3
Transmission: 2
Level: 0 (or 8)
Thresholds:
resistance 6: does more damage
transmission 6: gives the host minor armor

### PR DESCRIPTION
**Pull request by nikk735**
## About The Pull Request

adding new virus symptoms

## Why It's Good For The Game

resolves #754 

## Changelog
:cl:
add: Added new virus (Cornu Cutaneum) spiked skin
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Helped him code this, it's good. I'm averse to putting my own bounties to vote, but what the heck.

**Review by zeskorion — COMMENTED** [review]
Helped him code this, it's fine. I'm averse to putting my own bounties to vote, but what the heck.

**Comment by nikk735** [comment]
so will this be added?

**Comment by zeskorion** [comment]
> so will this be added?

just claim the bounty

**Comment by Bastian0930** [comment]
This specific PR keeps breaking my travis checks. Any idea why?

**Comment by nikk735** [comment]
It shouldn’t it runs fine while merged. So I assume the issue is on your side

File: code/datums/diseases/advance/symptoms/spiked.dm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+			if(prob(base_message_chance))
+				to_chat(M, "<span class='warning'> You see the spikes getting sharper and elongated.</span>")
+		if(5)
+			if(prob(base_message_chance))
+				to_chat(M, "<span class='warning'>You look like a human hedgehog</span>")
+
+/datum/symptom/spiked/End(datum/disease/advance/A)
+	var/mob/living/carbon/human/H = A.affected_mob
+	if(!..() && A.stage >= 2)
+		return
```
Comment (PowerfulBacon) [inline code/datums/diseases/advance/symptoms/spiked.dm:88]: makes a warning, and travis don't like warnings 😳

Comment (nikk735) [reply code/datums/diseases/advance/symptoms/spiked.dm:88]: oh zesk added the !..()  i dont even know what that truly means

**Pull request merged**